### DISCUSSION TITLE
OpenAPI (eka Swagger) Spec

### DISCUSSION THREAD

**Discussion by VinayaSathyanarayana** [keystonejs/keystone#7309] [Feature requests]
We should consider generating [OpenAPI (eka Swagger) Specs](https://www.openapis.org/) for the REST Api exposed for the models

**Comment by jesstelford** [comment]
There's not currently any plans to have REST endpoints, only GraphQL for the CRUD operations.

**Comment by bastiW** [comment]
Still no options for a REST API?

**Comment by jesstelford** [comment]
Keystone is unlikely to ever have a REST API, we've gone all in on GraphQL.

**Comment by bastiW** [comment]
Anyone tried to connect with the Prisma rest interface directly?

https://www.prisma.io/docs/concepts/overview/prisma-in-your-stack/rest

**Comment by junaid33** [comment]
It would be really nice to integrate https://www.sofa-api.com/. It takes a schema and creates a REST API. I haven't used it personally, but it would increase Keystone adoption if REST is an option.
